infra/build/functions/build_project.py

```diff
@@ -647,15 +656,19 @@ def create_config(args, build_type):
                 parallel=args.parallel,
                 upload=upload,
                 experiment=args.experiment,
-                build_type=build_type)
+                build_type=build_type,
+                fuzzing_engine=args.fuzzing_engine)
 
 
-def build_script_main(script_description, get_build_steps_func, build_type):
+def build_script_main(script_description,
+                      get_build_steps_func,
+                      build_type,
+                      args=None):
   """Gets arguments from command line using |script_description| as helpstring
   description. Gets build_steps using |get_build_steps_func| and then runs those
   steps on GCB, tagging the builds with |build_type|. Returns 0 on success, 1 on
   failure."""
-  args = get_args(script_description)
+  args = get_args(script_description, args)
```

Comment:
```suggestion
  description_args = get_args(script_description)  
  args.parse_args(description_args)
```

I don't know exactly the syntax, but the idea is doing something like this.